base/Dockerfile

```diff
@@ -21,7 +21,9 @@ RUN apt update -y && \
         wget \
         ca-certificates
 
-RUN TAR_PACKAGES="http://gca-jobs:1234/packages/libpimega_2.5.1-0_amd64.tar.gz" && \
+RUN TAR_PACKAGES="\
+        http://gca-jobs:1234/packages/libpimega_2.5.1-0_amd64.tar.gz \
+        http://gca-jobs:1234/packages/libsscpimega_0.8.12-0_amd64.tar.gz" && \
```

Comment:
Version should also be a variable. And libsscpimega should be built from source.

Reply:
It'll be built from source as soon as it becomes open source. :)

I'll include hard-coded versions in `.env` instead.

Reply:
We can still host a tarball at gca-jobs with the source. That way, we can deal with the necessary build (and deployment - if it's installed in `/usr/local`, it won't be copied automatically, so installing in `/opt` might make sense) steps before it's hosted on github.

Reply:
That's a good point. I may work on this build and installation process while still using the tarball. I may think more about it ans sketch somethings after working on other pendencies, such as script for installing tarballs and extracting the PRs for tarball install and `no-build` separately.